lms/djangoapps/instructor/utils.py

```diff
@@ -185,16 +183,15 @@ def collect_ora2_data(course_id):
 
 
 # pylint: disable=invalid-name
-def ora2_data_queries():
+def ora2_data_queries(include_email, course_id):
     """
     Wraps a raw SQL query which retrieves all ORA2 responses for a course.
     """
-
```

Comment:
Nit: I think we should keep this blank line to separate the multiline comment from the multiline string constant.